src/constructs/EcsServiceFactory.ts

```diff
@@ -215,10 +221,33 @@ export class EcsServiceFactory {
     }
 
     service.connections.securityGroups.forEach(sg => {
-      privateFileSystemSecurityGroup.addIngressRule(sg, Port.NFS);
+      securityGroup.addIngressRule(sg, Port.NFS);
     });
   }
 
+  private getImportedFileSystem() {
+    const securityGroup = this.getImportedFileSystemSecurityGroup();
+    if (!this.filesystem) {
+      const fileSystemArn = StringParameter.valueForStringParameter(this.scope, Statics._ssmFilesystemArn);
```

Comment:
Nu doen we hier hele specifieke dingen, kunnen we dit als parameters mee geven bij constructie van dit object? Wat als we nog een 2e filesystem nodig hebben later?

Reply:
Dat zou kunnen natuurlijk, maar de vraag is of je meer filesystems wil/gaat willen? Met de accesspoints kun je logisch onderscheid maken in welke applicatie welk deel van het filesystem kan gebruiken.